packages/next/server/image-optimizer.ts

```diff
@@ -602,6 +610,21 @@ export async function imageOptimizer(
       // End Squoosh transformation logic
     }
     if (optimizedBuffer) {
+      if (isDev && width === BLUR_IMG_SIZE && quality === BLUR_QUALITY) {
```

Comment:
Shouldn't this be checking either `width` or `height`?

Reply:
Hmm, you might have found a bug in the current dev server since we only pass width to the api. Let me add a test with a tall image and see what happens 👀

Reply:
Yeah this was a long standing bug where we hardcoded `w=8` for `next dev` placeholder.

Fixed in [fe74054](https://github.com/vercel/next.js/pull/39992/commits/fe74054d48ce7e904b0c9a38e289cf1733fb6448) and added a test 👍